octobot_commons/enums.py

```diff
@@ -64,6 +64,15 @@ class PriceIndexes(Enum):
     IND_PRICE_VOL = 5
 
 
+class PriceStrings(Enum):
```

Comment:
Why this is required to be in OctoBot-Commons ?

Reply:
Right, I moved it into the web interface enums because it's the only user so far.